core/src/main/java/io/neow3j/script/InteropService.java

```diff
@@ -17,8 +17,8 @@ public enum InteropService {
     SYSTEM_CONTRACT_CALL("System.Contract.Call", 1 << 15),
     SYSTEM_CONTRACT_CALLNATIVE("System.Contract.CallNative", 0),
     SYSTEM_CONTRACT_GETCALLFLAGS("System.Contract.GetCallFlags", 1 << 10),
-    SYSTEM_CONTRACT_CREATESTANDARDACCOUNT("System.Contract.CreateStandardAccount", 1 << 8),
-    SYSTEM_CONTRACT_CREATEMULTISIGACCOUNT("System.Contract.CreateMultisigAccount", 1 << 8),
+    SYSTEM_CONTRACT_CREATESTANDARDACCOUNT("System.Contract.CreateStandardAccount", 0),
```

Comment:
![Screenshot 2022-06-21 at 09 38 47](https://user-images.githubusercontent.com/37138571/174743535-f0a91f1d-ebdb-48c8-a1c9-5a42f13c38f6.png)
Based on the changes in the neo code, we can actually set the value to 1 << 15. See `CreateStandardAccount` in `Application.Contract.cs`. I was mistaken, it is still fixed after the hardfork. The value is according to the `SYSTEM_CRYPTO_CHECKSIG`. So we could use
```java
SYSTEM_CONTRACT_CREATESTANDARDACCOUNT("System.Contract.CreateStandardAccount", SYSTEM_CRYPTO_CHECKSIG.price)
```